composer.json

```diff
@@ -26,10 +25,14 @@
         "openeuropa/task-runner": "^0.8.0",
         "php-http/mock-client": "^1.0",
         "phpunit/phpunit": "~6.0",
+        "symfony/console": "^3.2 || ^4",
         "symfony/expression-language": "^4.2",
         "symfony/http-foundation": "~3.4",
         "zendframework/zend-code": "~3.3"
     },
+    "suggest": {
+        "symfony/console": "Allows use the command epoetry"
```

Comment:
Let's describe the dependency and its role in this project:
```
"symfony/console": "The Symfony Console component eases the creation of command line interfaces. Require this in your project if you wish to use the ePoetry Console."
```

Reply:
:+1: